module/documents/actor/trait.mjs

```diff
@@ -410,9 +414,20 @@ export function keyLabel(key, config={}) {
       if ( index ) return index.name;
       break;
     }
-  }
 
-  return key;
+    // Explicit categories (e.g. languages)
```

Comment:
Not clear to me how this differs from the above tool options where we search for an exact category match or a child match? Do we still need those, or are they covered by this new `searchCategory`?

Reply:
For vehicle categories in tools, those are stored in separate configuration objects, while these are nested categories in a single object. The joys of years of legacy configuration data :).

Reply:
Got it, thanks, that makes sense.